### PR TITLE
CQ-4290733 : [Visual Requirements-AEM Dynamic Media-Image Set Editor]…

### PR DESCRIPTION
**Pull request by shingari007**
…: Luminosity ratio fails for multiple controls, does not meets minimum luminosity ratio 4.5:1.

## Description
Expected Behavior

Multiple control coral button label and icon components should comply with WCAG 2.1 AA contrast ratio requirements with a contrast ratio between the foreground text color and the background color or at least 4.5:1.

Actual Behavior:

Currently, Coral-Spectrum uses var(--spectrum-global-color-gray-500); from Spectrum-CSS, for the ::coral label text and coral icon components, which results in contrast ratio less that 4.5:1, which does not comply with WCAG 2.1 AA contrast ratio requirements.

A simpler suggested fix would be to change the value in Coral-Spectrum to use var(--spectrum-global-color-gray-700); to satisfy the requirement.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: coral-component-button/src/styles/index.styl [hunk 1]
```diff
@@ -10,8 +10,27 @@
...
+
 // Avoid FOUC
 button[is="coral-button"] {
   background-color: transparent;
   border-color: transparent;
 }
+
+.coral--light ._coral-ActionButton--quiet:disabled,
+.coral--light ._coral-ActionButton--quiet.is-disabled,
+.coral--light ._coral-Tool:disabled {
```
Comment (mohiaror) [inline coral-component-button/src/styles/index.styl:23]: Do we need to update the CSS for `coral-Tool`? Why is it required? Same for all other places.

Comment (shingari007) [reply coral-component-button/src/styles/index.styl]: I have revisited the changes.
Yes we do not need to update the css for _coral-Tool.
I have done the changes accordingly.

**Comment by majornista** [comment]
@shingari007 You need to sign the Adobe CLA.

**Comment by shingari007** [comment]
> @shingari007 You need to sign the Adobe CLA.

done

**Pull request merged**